### ISSUE TITLE
Support for octave 4.2.1 on Mac OSX 10.12

### ISSUE THREAD

**Issue by roversch** [casadi/casadi#2000]
After updating to OSX 10.12, homebrew automatically upgraded my `octave` installation from `4.2.0` to `4.2.1`. However, this version seems not to be supported by `casadi-3.1.1`:

```
error: sym: .../casadi-octave-v3.1.1/casadiMEX.mex: failed to load: 

dlopen(.../casadi-octave-v3.1.1/casadiMEX.mex, 10): Library not loaded: 
/usr/local/opt/octave/lib/octave/4.2.0/liboctinterp.4.dylib
  
  Referenced from: .../casadi-octave-v3.1.1/casadiMEX.mex
  Reason: image not found
```

I first tried to switch versions with `brew switch`, but I also needed to switch many dependencies of `octave` to an earlier version, and I'm not sure if that might break something else.

**Comment by roversch** [comment]
For a temporary workaround, I was looking for a build with the new octave: [this](https://travis-ci.org/casadi/binaries/builds/207086687#L3276 ) is the most recent successful binary build on OSX with `octave-4.2.1`. However, I could not find it on sourceforge, I guess old binaries get deleted, right?

**Comment by jgillis** [comment]
Now available with 3.2.0-rc1
